src/styles/blocks/stars.css

```diff
@@ -0,0 +1,36 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../../images/star.svg);
+  height: 16px;
+  width: 16px;
+
+  background-repeat: no-repeat;
+  margin-right: 4px;
+  background-position: center;
+
+  margin-bottom: 24px;
+}
+
+.stars--1 .stars__star:nth-child(-n+1),
+.stars--2 .stars__star:nth-child(-n+2),
+.stars--3 .stars__star:nth-child(-n+3),
+.stars--4 .stars__star:nth-child(-n+4),
+.stars--5 .stars__star:nth-child(-n+5) {
+  background-image: url(../../images/star-active.svg);
+}
+
+.stars__review {
+  font-weight: 400;
```

Comment:
default

Reply:
I set in the html the font-weight to 500, that's why it was there. But now removed it.